src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,15 +1,38 @@
+import createFetch from '@vercel/fetch'
 import NextAuth from 'next-auth'
 import CloudIdpProvider from 'lib/auth/cloud-idp-provider'
 import refreshTokenSet from 'lib/auth/refresh-token-set'
 import isJwtExpired from 'lib/auth/is-jwt-expired'
 import { AuthErrors } from 'types/auth'
 
+const fetch = createFetch()
+
 const isDev = process.env.NODE_ENV === 'development'
 
 export default NextAuth({
 	session: {
 		maxAge: 2592000, // 30 days
 	},
+	// https://next-auth.js.org/configuration/options#events
+	events: {
+		/**
+		 * NOTE: NextAuth does not log out of auth providers, so we have to handle doing that
+		 * ourselves in this signOut event.
+		 * https://github.com/nextauthjs/next-auth/discussions/3938
+		 */
+		async signOut({ token }) {
+			try {
+				await fetch(
```

Comment:
I don't see this fetch when logging out with the vercel app. Do you know why that may be?

Reply:
If you mean in your browser's network inspector, I believe this network request is server-side.

Reply:
Ah! I didn't realize that this was server side. Thank you for clarifying.

Reply:
Can confirm! This request happens server-side